packages/react-dom/src/events/DOMPluginEventSystem.js

```diff
@@ -201,6 +201,9 @@ export const mediaEventTypes: Array<DOMEventName> = [
   'waiting',
 ];
 
+// List of events that need to be individually attached to video elements.
+export const videoEventTypes: Array<DOMEventName> = ['resize'];
```

Comment:
Note: I created a separate `videoEventTypes` array, since I doubt anyone will want to add `onResize` to an audio event. It would simplify the code a bit to just add `resize` to the `mediaEventTypes` array, if that’s preferred.

Reply:
A one-item array seems like overkill to me. Are there any downsides to adding it to `mediaEventTypes`? I'd suggest that.

Reply:
Happy to make the change! The only “downside” is that since [the `resize` event spec](https://html.spec.whatwg.org/multipage/media.html#event-media-resize) lists:

> Media element is a `video` element

in its preconditions, we _might_ want to log a warning if someone attaches `onResize` to an `audio` element. But I agree, I think the simplification is worth it. I’ll make the change now.

Reply:
Done ✨